catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/query/GazetteerQueryCatalog.java

```diff
@@ -181,18 +181,21 @@ public GeoEntry queryById(String id) throws GeoEntryQueryException {
     suggestProps.put(SUGGESTION_CONTEXT_KEY, GAZETTEER_METACARD_TAG);
     suggestProps.put(SUGGESTION_DICT_KEY, SUGGEST_PLACE_KEY);
 
+    SortBy[] sortbys = {
+      new SortByImpl(GeoCodingConstants.GAZETTEER_SORT_VALUE, SortOrder.ASCENDING)
```

Comment:
❓ should `GeoCodingConstants.COUNTRY_GAZETTEER_SORT_VALUE` also be added as a secondary sort value to this list?